metric-anomaly-detector/src/test/java/org/hypertrace/alert/engine/metric/anomaly/detector/MetricAnomalyDetectorTest.java

```diff
@@ -1,20 +1,14 @@
 package org.hypertrace.alert.engine.metric.anomaly.detector;
 
-import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import static org.hypertrace.alert.engine.metric.anomaly.detector.AlertRuleEvaluatorTest.createLeafFilter;
+import static org.hypertrace.alert.engine.metric.anomaly.detector.AlertRuleEvaluatorTest.createLhsExpression;
```

Comment:
Can/should we create a separate utility class for these functions ?

Reply:
We do it at a later point